### PR TITLE
Exclusion tables: merged tables

### PR DESCRIPTION
**Pull request by joeflack4**
Updates
```
Exclusion tables
- Added feature to merge tables together
```

### CODE REVIEW

File: src/ontology/mondo-ingest.Makefile [hunk 1]
```diff
@@ -181,7 +181,11 @@ config/%_term_exclusions.txt config/%_exclusion_reasons.robot.template.tsv: conf
 	--component-signature-path $(word 4,$^) \
 	--config-path $(word 5,$^)
 
-all-exclusions: config/ordo_term_exclusions.txt config/doid_term_exclusions.txt config/omim_term_exclusions.txt config/icd10cm_term_exclusions.txt config/icd10who_term_exclusions.txt config/ncit_term_exclusions.txt
+# Combines all tables
+config/term_exclusions.txt config/exclusion_reasons.robot.template.tsv: config/ordo_term_exclusions.txt config/doid_term_exclusions.txt config/omim_term_exclusions.txt config/icd10cm_term_exclusions.txt config/icd10who_term_exclusions.txt config/ncit_term_exclusions.txt
```
Comment (joeflack4) [inline src/ontology/mondo-ingest.Makefile:185]: The new merge goal / command.

Comment (joeflack4) [inline src/ontology/mondo-ingest.Makefile:185]: This is ready to merge if we want the TSV / TXT stuff now.

I'm unclear on (i) docs, and (ii) OBO/TTV. I put my questions here: https://github.com/monarch-initiative/mondo-ingest/issues/54

File: src/ontology/mondo-ingest.Makefile [hunk 2]
```diff
@@ -181,7 +181,11 @@ config/%_term_exclusions.txt config/%_exclusion_reasons.robot.template.tsv: conf
...
+	@rm config/term_exclusions.txt; rm config/exclusion_reasons.robot.template.tsv; \
+	cat config/*_term_exclusions.txt > config/term_exclusions.txt; \
+	awk '(NR == 1) || (NR == 2) || (FNR > 2)' config/*_exclusion_reasons.robot.template.tsv > config/exclusion_reasons.robot.template.tsv
```
Comment (matentzn) [inline src/ontology/mondo-ingest.Makefile:188]: LOL hahha. Great.

**Review by matentzn — APPROVED** [review]
Great this can be merged if this works inside ODK (`sh run.sh make...`)

**Comment by joeflack4** [comment]
Yep, it does. So far I haven't encountered many issues (if any) using the docker container to run these `make` commands. I'll go ahead and merge.

**Pull request merged**